### PR TITLE
Update gh-pages for 21.06 release

### PR DESCRIPTION
**Pull request by sameerz**
Update gh-pages for 21.06 release.

### CODE REVIEW

File: docs/additional-functionality/cache-serializer.md [hunk 1]
```diff
@@ -29,22 +29,30 @@ nav_order: 2
...
   | ------ | -----|
   | 3.1.1 | com.nvidia.spark.rapids.shims.spark311.ParquetCachedBatchSerializer |
   | 3.1.2 | com.nvidia.spark.rapids.shims.spark312.ParquetCachedBatchSerializer |
-  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer |
+  | 3.1.3 | com.nvidia.spark.rapids.shims.spark313.ParquetCachedBatchSerializer |
+  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer | 
+  
+##          Supported Types                       
+ 
+ All types are supported on the CPU, on the Gpu, ArrayType, MapType and BinaryType are not
```
Comment (GaryShen2008) [inline docs/additional-functionality/cache-serializer.md:55]: Should "Gpu" be changed to "GPU"?

**Review by GaryShen2008 — APPROVED** [review]

File: docs/additional-functionality/cache-serializer.md [hunk 2]
```diff
@@ -29,22 +29,30 @@ nav_order: 2
...
   | ------ | -----|
   | 3.1.1 | com.nvidia.spark.rapids.shims.spark311.ParquetCachedBatchSerializer |
   | 3.1.2 | com.nvidia.spark.rapids.shims.spark312.ParquetCachedBatchSerializer |
-  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer |
+  | 3.1.3 | com.nvidia.spark.rapids.shims.spark313.ParquetCachedBatchSerializer |
+  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer | 
+  
+##          Supported Types                       
+ 
+ All types are supported on the CPU, on the GPU, ArrayType, MapType and BinaryType are not
```
Comment (jlowe) [inline docs/additional-functionality/cache-serializer.md:55]: The fix from Gpu -> GPU is not on branch-21.06.  Normally these kind of PRs don't fixup docs, they should just merge the changes in the docs to the gh-pages branch.  Patching things "on the fly" in these PRs can easily leave those same fixes missing from the development branches, and anyone looking at the documentation on those branches will not see those fixes.  There should be a followup to fix this in branch-21.08.

**Review by jlowe — CHANGES_REQUESTED** [review]
docs/FAQ.md changes are missing

File: docs/additional-functionality/cache-serializer.md (hunk 2 above)
Comment (sameerz) [reply docs/additional-functionality/cache-serializer.md:55]: Added PR #2840 for the documentation fix.

**Comment by sameerz** [comment]
Added FAQ.md.

**Review by jlowe — APPROVED** [review]

**Pull request merged**